contrib/gtktheme/jgmenu-gtktheme.py

```diff
@@ -56,9 +56,11 @@ def process_line(line):
         setconfig("color_title_border", rgb2hex(line))
 
 def cache(themename):
-    """ save the theme-name to ~/.cache/jgmenu/.last-gtktheme """
+    """ save the theme-name to XDG_CACHE_HOME/jgmenu/.last-gtktheme """
+    from xdg.BaseDirectory import xdg_cache_home
```

Comment:
Importing from xdg crashes for me - I obviously do not have the xdg module installed
In order to avoid creating work for distro maintainers (@johnraff ??) I prefer going manual on this one.
Any chance of manually using `XDG_CACHE_HOME` if set/non-empty or else use `~/.cache`

```
Traceback (most recent call last):
  File "/home/johan/tmp/jgmenu/./contrib/gtktheme/jgmenu-gtktheme.py", line 106, in <module>
    main()
  File "/home/johan/tmp/jgmenu/./contrib/gtktheme/jgmenu-gtktheme.py", line 74, in main
    cache(themename)
  File "/home/johan/tmp/jgmenu/./contrib/gtktheme/jgmenu-gtktheme.py", line 60, in cache
    from xdg.BaseDirectory import xdg_cache_home
ModuleNotFoundError: No module named 'xdg'
```

Reply:
XDG_CACHE_HOME is unset by default in Debian, so a test to only use that envvar if it is set, otherwise reverting to the default $HOME/.cache, would be desirable IMO.